mypy.ini

```diff
@@ -74,7 +74,7 @@ ignore_errors = True
 ignore_errors = False
 
 [mypy-manim.mobject.geometry.*]
-ignore_errors = True
+ignore_errors = False
```

Comment:
PR #4306 changed the `mypy.ini` file from a blacklist to a whitelist: all errors are now raised by default and you must explicitly add a file to it in order to ignore its errors.

Therefore, instead of adding a `[[mypy-manim.mobject.geometry.*]` entry to stop ignoring errors, now you have to *remove* the already existing entry from the current `mypy.ini` in order to stop ignoring errors.